test/services/most_popular_content_test.rb

```diff
@@ -14,7 +14,7 @@ def taxon_content_id
   end
 
   describe '#fetch' do
-    it 'returns the results from search, sorted by title' do
+    it 'returns the results from search' do
```

Comment:
Given the commit message we're not testing what the amended ordering is here - not sure if this is covered elsewhere or in another commit.